client/src/components/Cards/Link/index.js

```diff
@@ -95,8 +95,8 @@ const Link = ({
               <T.P color="gray8" caps>
                 {programme?.date ? dateFormatter(programme.date) : 'N/A'}
               </T.P>
-              <T.P weight="light" mr="1">
-                View <span style={{ fontWeight: 'bold' }}>programme</span>
+              <T.P weight="bold" mr="1">
+                {programme?.title}
```

Comment:
`?.` will return the title if exists and return null if its not, for that please provide a default value to cover that case 
```suggestion
                {programme?.title || 'N/A'}
```